extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -264,6 +254,37 @@ class QueueProcessor {
         });
     }
 
+    _setupClients(sourceRole, targetRole) {
```

Comment:
I agree, I did this just to have a working mvp. Once, we have a working copy we can cleanup the setup.